book/src/SUMMARY.md

```diff
@@ -0,0 +1,9 @@
+# Summary
+
+- [Introduction](./introduction.md)
+
```

Comment:
Does it make sense to link the rustdoc here too?

Reply:
Yeah I’ve been wondering what’s best. Either link from the book (can’t add external links in the summary though I believe) or have a main page that directs you to spec, rustdoc, etc.